pkg/oc/cli/newapp/newapp.go

```diff
@@ -1019,7 +1019,15 @@ func TransformRunError(err error, baseName, commandName, commandPath string, gro
 		// TODO: suggest things to the user
 		groups.Add("", "", "", UsageError(commandPath, newAppNoInput, baseName, commandName))
 	default:
-		groups.Add("", "", "", err)
+		if runtime.IsNotRegisteredError(err) {
+			useProcessMsg := fmt.Sprintf("The template contained an object type unknown to `oc new-app`.  Use `oc process | oc create` instead.  The decoder returned: %s", err.Error())
```

Comment:
Change to `Error details: %v` instead of `The decoder returned:` user doesn't and will not know what's a decoder. Let's not confuse them. Ditto below.

Reply:
change pushed @soltysh